### DISCUSSION TITLE
blog/errores-comunes-front-end

### DISCUSSION THREAD

**Discussion by giscus** [adrianRacovitaJSX/portfolio#2] [General]
# blog/errores-comunes-front-end

Descubre los errores más comunes que muchos desarrolladores front-end cometemos al iniciar y cómo aprender de ellos puede acelerar tu crecimiento profesional.

http://localhost:3000/blog/errores-comunes-front-end

**Comment by adrianRacovitaJSX** [comment]
Comentario
